src/main/java/hudson/plugins/release/pipeline/ReleaseStep.java

```diff
@@ -0,0 +1,218 @@
+package hudson.plugins.release.pipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.Nonnull;
+
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepExecutionImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.jenkinsci.plugins.workflow.util.StaplerReferer;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.DoNotUse;
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
+
+import com.google.inject.Inject;
+
+import hudson.AbortException;
+import hudson.Extension;
+import hudson.console.ModelHyperlinkNote;
+import hudson.model.AbstractProject;
+import hudson.model.Action;
+import hudson.model.AutoCompletionCandidates;
+import hudson.model.BuildableItemWithBuildWrappers;
+import hudson.model.Cause;
+import hudson.model.ItemGroup;
+import hudson.model.Job;
+import hudson.model.ParameterDefinition;
+import hudson.model.ParameterValue;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.model.queue.QueueTaskFuture;
+import hudson.plugins.release.ReleaseWrapper;
+import hudson.plugins.release.SafeParametersAction;
+import jenkins.model.Jenkins;
+import jenkins.model.ParameterizedJobMixIn;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+
+/**
+ * Created by e3cmea on 1/3/17.
+ *
+ * @author Alexey Merezhin
+ */
+public class ReleaseStep extends AbstractStepImpl {
+    private static final Logger LOGGER = Logger.getLogger(ReleaseStep.class.getName());
+
+    private String job;
+    private List<ParameterValue> parameters;
+
+    @DataBoundConstructor
+    public ReleaseStep(String job) {
+        this.job = job;
+        parameters = new ArrayList<>();
+    }
+
+    public String getJob() {
+        return job;
+    }
+
+    public void setJob(String job) {
+        this.job = job;
+    }
+
+    public List<ParameterValue> getParameters() {
+        return parameters;
+    }
+
+    @DataBoundSetter public void setParameters(List<ParameterValue> parameters) {
```

Comment:
FYI in the [pipeline dev guide](https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md) annotation is on the same line:
```java
@DataBoundSetter public void setStuff(@CheckForNull String stuff) {
    this.stuff = Util.fixNull(stuff);
}
```